examples/30_extended/fetch_runtimes_tutorial.py

```diff
@@ -325,3 +357,73 @@ def get_incumbent_trace(trace):
 plt.title("Optimisation Trace Comparison")
 plt.legend()
 plt.show()
+
+################################################################################
+# Case 4: Running models that scikit-learn doesn't parallelize
+# *************************************************************
+# Both scikit-learn and OpenML depend on parallelism implemented through `joblib`.
+# However, there can be cases where either models cannot be parallelized or don't
+# depend on joblib for its parallelism. 2 such cases are illustrated below.
+#
+# Running a Decision Tree model that doesn't support parallelism implicitly, but
+# using OpenML to parallelize evaluations for the outer-cross validation folds.
+
+dt = DecisionTreeClassifier()
+run6 = openml.runs.run_model_on_task(
+    model=dt, task=task, upload_flow=False, avoid_duplicate_runs=False, n_jobs=2
+)
+measures = run6.fold_evaluations
+print_compare_runtimes(measures)
+
+################################################################################
+# Running a Neural Network from scikit-learn that uses scikit-learn independent
+# parallelism using libraries such as MKL, OpenBLAS or BLIS.
+
+mlp = MLPClassifier(max_iter=10)
+run7 = openml.runs.run_model_on_task(
+    model=mlp, task=task, upload_flow=False, avoid_duplicate_runs=False
+)
+measures = run7.fold_evaluations
+print_compare_runtimes(measures)
+
+################################################################################
+# Summmary
+# *********
+# OpenML records model runtimes for the CPU-clock and the wall-clock times. The above
```

Comment:
Hm, should we explicitly say "the scikit-learn extension"? Everything we're writing here is exclusive about the scikit-learn extension, so it could be confusing otherwise.